source/extensions/filters/http/cdn_loop/filter.cc

```diff
@@ -0,0 +1,48 @@
+#include "extensions/filters/http/cdn_loop/filter.h"
+
+#include "envoy/http/codes.h"
+#include "envoy/http/filter.h"
+#include "envoy/http/header_map.h"
+
+#include "common/common/statusor.h"
+
+#include "extensions/filters/http/cdn_loop/utils.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace CdnLoop {
+
+namespace {
+const Http::LowerCaseString CdnLoopHeaderName("CDN-Loop");
```

Comment:
Paging @mattklein123  - my preference would be to stick this in Headers::HeaderEntries, but there's no way to do that without incurring non-guarded L7 changes :-/   Thoughts?

Reply:
This is gone now that the filter uses CustomInlineHeaders.